std_msgs/Header header # Standard header information, including timestamp and frame ID
string text # Content of speech recognition
float64 yaw # Direction vector of sound